packages/metro/src/ModuleGraph/worker/__tests__/collectDependencies-test.js

```diff
@@ -32,94 +32,24 @@ const nullthrows = require('nullthrows');
 
 const {any, objectContaining} = expect;
 
-const {InvalidRequireCallError, DefaultModuleDependencyRegistry} =
-  collectDependencies;
+const {InvalidRequireCallError} = collectDependencies;
 const opts = {
   asyncRequireModulePath: 'asyncRequire',
   dynamicRequires: 'reject',
   inlineableCalls: [],
   keepRequireNames: true,
   allowOptionalDependencies: false,
   dependencyMapName: null,
+  unstable_allowRequireContext: true,
```

Comment:
1. Let's set this to `false` for the test and then override it for the `require.context` tests. (I think we're actually doing that in the test below.)
2. We could use a test of what happens when `require.context` is found in the input when this is `false` - essentially a test that this feature isn't accidentally on for everyone. (We could later also implement a nice error message saying that this feature isn't enabled, and have that reflected in the test.)